lib/measures/tbd/measure.rb

```diff
@@ -1,4 +1,13 @@
-require_relative "tbd"
+require_relative "resources/version"
+require_relative "resources/geometry"
+require_relative "resources/transformation"
+require_relative "resources/model"
+
+require_relative "resources/psi"
+require_relative "resources/conditioned"
+require_relative "resources/framedivider"
+require_relative "resources/ua"
+require_relative "resources/log"
 
```

Comment:
Subfolders are NOT allowed under /resources - only files, as described [here](https://nrel.github.io/OpenStudio-user-documentation/reference/measure_writing_guide/) (search 'Measure resource files'). None of the spec tests had previously failed (CLI) - only post-assembly App tests when running simulations (measure.xml holding errors on missing folders/files). Surprisingly, no App issues running the measure under 'Apply Measures Now'.

Involves changes to Rakefile, and measure.rb.

Reply:
You could try to include tbd here (in same the gem is available) and then fall back requiring from resources, but that is up to you